std/php7/Lib.hx

```diff
@@ -85,12 +85,12 @@ class Lib {
 		return untyped __call__("fpassthru", __call__("fopen", file,  "r"));
 	}
 
-	public static function toPhpArray(a : Array<Dynamic>) : NativeArray {
-		throw "Not implemented";
+	public static inline function toPhpArray(a : Array<Dynamic>) : NativeArray {
```

Comment:
I have to try if something can go wrong if you apply this to an already casted type (if you can do this at all, not sure).
Basically simple casting should be encouraged.